docs/dev/new_check_howto.md

```diff
@@ -300,9 +300,59 @@ The `logos` folder must contain **three** images with filenames and sizes that m
+| `package_deps`      | String          | Optional           |                                                                                                                                                                                                                                    |
+| `use_omnibus_reqs`  | Boolean         | Optional           |                                                                                                                                                                                                                                    |
+| `metric_to_check`   | String          | Optional           | The presence of this metric is the determiner of if this integration is working properly. If this metric is not being reported when this integration is installed, the integration is marked as broken in the Datadog application. |
+| `metric_prefix`     | String          | Optional           | The namespace for this integration's metrics. Every metric reported by this integration will be prepended with this value.                                                                                                         |
+| `version`           | String          | Optional           |                                                                                                                                                                                                                                    |
+
+
+#### metadata.csv
+
+Our example check doesn't send any metrics, so in this case we will leave it empty but find below the description for each column of your `metadata.csv` file:
+
+| Column name     | Mandatory/Optional | Description                                                                                                                                                                     |
+| ---             | ----               | ----                                                                                                                                                                            |
+| `metric_name`   | Mandatory          | Name of the metric.                                                                                                                                                             |
+| `metric_type`   | Mandatory          | [Type of the metric][10].                                                                                                                                                       |
+| `interval`      | Optional           | Interval of collection of the metric in second.                                                                                                                                 |
```

Comment:
`Collection interval` ?

Reply:
@l0k0ms ^

Reply:
The `metadata.csv` files column name is `interval`, but I updated the description.